app/scripts/lib/metrics.js

```diff
@@ -303,6 +305,18 @@ define(function (require, exports, module) {
     },
 
     /**
+     * Log an event only if it never happened before during this page load.
+     *
+     * @param {String} eventName
+     */
+    logEventOnce: function (eventName) {
+      if (eventMemory.indexOf(eventName) === -1) {
```

Comment:
That was bad vocabulary on my part, what I really meant was to use one of these: `{}`.